sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/SyncAccountRepository.kt

```diff
@@ -554,6 +562,7 @@ sealed class Result<out R> {
     data class Error(
         val code: Int = GENERIC_ERROR.code,
         val reason: String = "",
+        val payload: Any? = null,
```

Comment:
is there no way to better _type_ this? if not, it would be good to at least document the data class to indicate what `payload` can be used for.

Reply:
There's a way, to remove it :) It was a leftover from a previous approach, that I ended up removing because I didn't like the payload there.